### DISCUSSION TITLE
General questions from workshop

### DISCUSSION THREAD

**Discussion by mxochicale** [SciKit-Surgery/sustainable-pkg-stats#27] [Q&A]
* Can we customise search words after creating template? 
* Can we use template with private repositories?
* What's the aim of the token? We can create a classic token than a fine-grain token

**Comment by thompson318** [comment]
>     * What's the aim of the token? We can create a classic token than a fine-grain token
Token should be sorted now with #29. Token is only necessary for running scripts locally.
